### DISCUSSION TITLE
cq码上传图片有错误，请大佬们看一下嗷

### DISCUSSION THREAD

**Discussion by xia2233** [Mrs4s/go-cqhttp#1490] [Q&A] [answered]
![屏幕截图 2022-05-05 012044](https://user-images.githubusercontent.com/104424500/166743834-06f29b55-fba7-41fc-9031-4bd5db0a897e.jpg)

**Comment by xia2233** [comment, marked as answer]
已经自己搞定了哈哈哈哈

**Reply by Jodesen** [reply]
怎么搞定的，好像我也遇到了/(ㄒoㄒ)/~~
